Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -35,5 +265,17 @@ public AuthorizationClient(HttpClient client)
             var error = JsonConvert.DeserializeObject<Error>(stringResponse);
             throw new AuthorizationException(error);
         }
+
+        private async Task SendRequest(HttpRequestMessage message)

```

Comment:
What happens when we have a 500 error? Are we able to Deserialize that response into an Error object?

Reply:
Added logic to handle this.